third-party/utilities/py/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+cmake_minimum_required(VERSION 3.1.0)
+project( pyrsutils )
+
+set( DEPENDENCIES utilities )
+
+set( PYRSUTILS_FILES
+    "pyrsutils.cpp"
+    )
+
+pybind11_add_module( ${PROJECT_NAME} SHARED ${PYRSUTILS_FILES} )
```

Comment:
I think we better add 
```
target_compile_definitions(${PROJECT_NAME} PRIVATE -DPYBIND11_COMPILER_TYPE="_rsutils_abi" )
```
and maybe update on [external_pybind11.cmake](https://github.com/IntelRealSense/librealsense/pull/11147/files#diff-54f5d85af3be293a146e98b92de9b7f25f6aa5dc645a75d6003a9e7b83afebb5) to project specific definition of:
```
target_compile_definitions(${PROJECT_NAME} PRIVATE -DPYBIND11_COMPILER_TYPE="_librs_abi" )
```

otherwise bad things can heppen when running multiply pybind11 libraries (this was written in blood)

Reply:
Pybind is only compiled once.
Adding it here won't do anything -- it's set once for all our modules.
The ABI needs to be specific to the MSVC compiler, and the chance that our libraries are compiled with different MSVC is nonexistent...